pkg/rancher-desktop/main/commandServer/__tests__/settingsValidator.spec.ts

```diff
@@ -83,6 +83,7 @@ describe(SettingsValidator, () => {
         'virtualMachine.hostResolver':             'win32',
         'virtualMachine.memoryInGB':               'darwin',
         'virtualMachine.numberCPUs':               'linux',
+        'kubernetes.ingress.localhostOnly':        'win32',
```

Comment:
I just sorted all keys alphabetically in a previous PR; can we keep the sort order?